sources/Cargo.lock

```diff
@@ -94,9 +94,9 @@ dependencies = [
 
 [[package]]
 name = "actix-http"
-version = "3.8.0"
+version = "3.9.0"
```

Comment:
Is this large dependency update necessary? I'd prefer something more targeted to just the settings SDK update.

Reply:
`cargo build` inside sources directory instead of `cargo update` worked for me. iirc, I had to manually change the references to `settings-sdk` to my branch but there might be better ways to do it.

Reply:
I used `cargo check` to regenerate the file.

Reply:
TIL, if I delete the `Cargo.lock` file and regenerate it with `cargo check`, the dependencies might get updated.